lib/src/components/accordion/accordion.dart

```diff
@@ -0,0 +1,85 @@
+import 'package:flutter/material.dart';
+
+import '../../../flutter_dsfr.dart';
+import 'accordion_border.dart';
+import 'accordion_panel.dart';
+
+/// Create an Accordion .
+/// Must be render inside a Scrollable widget
+///
+/// Specs: https://gouvfr.atlassian.net/wiki/spaces/DB/pages/312082509/Accord+on+-+Accordion
+class DSFRAccordion extends StatefulWidget {
+  const DSFRAccordion({required this.panels, super.key});
+
+  final List<DSFRAccordionData> panels;
+
+  @override
+  State<StatefulWidget> createState() => _DSFRAccordionState();
+}
+
+class _DSFRAccordionState extends State<DSFRAccordion> {
+  UniqueKey? accordionValue;
+  late final itemsValues = <UniqueKey>[];
```

Comment:
You could directly use `List<UniqueKey>.generate` instead of relying on `_initItemsValues`:
```dart
late final _itemsValues = List<UniqueKey>.generate(widget.panels.length, (_) => UniqueKey());
```

Reply:
looks good